boards/px4/fmu-v6x/default.px4board

```diff
@@ -20,6 +20,7 @@ CONFIG_DRIVERS_IMU_BOSCH_BMI088=y
 CONFIG_DRIVERS_IMU_INVENSENSE_ICM20602=y
 CONFIG_DRIVERS_IMU_INVENSENSE_ICM20649=y
 CONFIG_DRIVERS_IMU_INVENSENSE_ICM20948=y
+CONFIG_DRIVERS_IMU_INVENSENSE_ICM42670P=y
```

Comment:
FYI the icm42670p was a community contribution with some differences we need to fix. For example it still has the sensor side filter (AKA DLPF) enabled at 34 Hz. https://github.com/PX4/PX4-Autopilot/blob/497d39621d6aa1f8a175b5e59b9ab6cd3f5befe2/src/drivers/imu/invensense/icm42670p/ICM42670P.hpp#L178-L179

I'll do a thorough review with updates once I have hardware in hand.

Reply:
Sure, I will send the hardware over once available.